content/en/docs/setup/upgrade/canary/revision_tags_test.sh

```diff
@@ -56,5 +56,7 @@ _verify_contains snip_usage_8 "istiod-$fullVersionRevision"
 
 # @cleanup
 snip_uninstall_old_control_plane_1
+kubectl delete mutatingwebhookconfiguration istio-revision-tag-prod-canary
```

Comment:
why is this not part of uninstall? is it a new change coming in? If so can you point to the istio PR which changes this?

Reply:
It's not a new change: a tag webhook should be created with `istioctl tag set` and removed with `istioctl tag remove`, since these are user-created items not generated by the installer. `istioctl uninstall` should not remove these, to avoid issues like https://github.com/istio/istio/issues/47423.

Reply:
was wondering why the test started failing all of a sudden now